doc/data/messages/r/redundant-unittest-assert/bad.py

```diff
@@ -4,3 +4,8 @@
 class DummyTestCase(unittest.TestCase):
     def test_dummy(self):
         self.assertTrue("foo")  # [redundant-unittest-assert]
+
+
+def test_division():
+    a = 9 / 3
+    assert "No ZeroDivisionError were raised"  # assert-on-string-literal
```

Comment:
Why are we adding an example for another message here?

Reply:
I added an example expecting this code to raise a ``[redundant-unittest-assert]``, but I think those two messages should be the same one. I think it can illustrate the two ways to fix the issue.

Reply:
But this example is about `unittest` assertions, not normal `assert`. I don't think we should merge those.